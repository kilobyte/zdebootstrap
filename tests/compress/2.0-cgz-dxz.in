Format: 2.0
CZ: gz
DZ: xz
