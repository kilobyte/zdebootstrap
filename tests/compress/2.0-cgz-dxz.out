
usr
usr/share
usr/share/doc
usr/share/doc/dupa
usr/share/doc/dupa/README
var
var/lib
var/lib/dpkg
var/lib/dpkg/info
var/lib/dpkg/info/dupa.list
var/lib/dpkg/status
